exercises/APE8/APE08-E3.tex

```diff
@@ -0,0 +1,76 @@
+\subsection*{a.}
+To show that this game allows a correlated equilibrium for which each player receives a payoff higher than 4. We can see that if we take same correlated strategies for all the payoff non zero, we have a payoff higher than 4. So we have :
+$$ \mu(c_1,c_2) = \left\{ 
```

Comment:
There is the `cases` environment that is useful for that ;) https://tex.stackexchange.com/questions/321660/how-to-use-the-cases-environment